site/content/entry/yuzu-progress-report-oct-2021/index.md

```diff
@@ -0,0 +1,430 @@
+
+If the GPU runs out of VRAM faster than the emulator or the driver can handle it, yuzu will close, so follow our recommended values in the picture below.
+
+{{< imgs
+	"./vram.png|We recommend 2GB and lower users to use 1X on graphic intensive games"
+>}}
+
+Small low-end GPUs (like our recommended GT 1030, RX 550, or any integrated GPU) will see considerable performance benefits when upsampling as the reduction in 3D load and the 
+lower VRAM utilization will provide higher framerates.
+
+People with current mid to high-end GPUs can benefit from the surplus of computational capabilities and available VRAM. 
+Actual performance will be determined by each specific game, but it’s safe to assume that good hardware can use a 2X or 3X multiplier perfectly fine most of the time.
+We personally don’t recommend going higher than 3X as not even the best consumer hardware available at this moment can provide a perfect experience on very high rendering 
+resolutions, but the options are there to experiment with. 
+
+Light or very well optimized games like `Super Smash Bros. Ultimate` or `Metroid Dread` can make for great experiences on 4K or 8K displays.
+On the other hand, particular games like `Astral Chain` should not be downsampled, as they already use 3840x2160 textures. 
+Trying high resolution multipliers will result in your VRAM not being very happy.
+
+We strongly recommend using Vulkan for downsampling as VRAM use will be considerably lower compared to OpenGL.
+
+Let’s move on to the second option, `Window Adapting Filter`.
+
+When you have to display on a different resolution as your rendering, you need some way of deciding which pixels will be shown in the final result, filters are used for that.
+
+At the moment yuzu offers 6 filtering options, each recommended for different scenarios, and with different performance costs. We plan to expand this section in the future, 
+adding sliders to let users customize the filters to their own tastes.
+And yes, on GPU limited scenarios, the filter in use can affect framerate.
+
+{{< imgs
+	"./filters.png|Tested on Mario Kart 8 Deluxe, filtering ain't free!"
+>}}
+
+While an integrated GPU will suffer the performance cost, even a low end dedicated card will power through with ease.
+
+Here are some recommendations and comparisons of all current options against the default bilinear.
+
+`Nearest Neighbour` is ideal for pixel art style games like `UNDERTALE` or `OCTOPATH TRAVELER`, it can also provide the best results when the game’s rendering resolution 
+matches the window or display’s resolution. Has the lowest performance cost of all options.
+
+{{< single-title-imgs-compare
+	"OCTOPATH TRAVELER (Bilinear Vs. Nearest Neighbour)"
+	"./otbilinear.png"
+	"./otnearest.png"
+>}}
+
+`Bilinear` is our combat tested default option, very good for downsampling and like nearest neightbour, it has the lowest performance cost, it’s a great all-rounder.
```

Comment:
```suggestion
`Bilinear` is our combat tested default option. It's very good for downsampling and like `Nearest Neighbor`, it has a very low performance cost. It’s a great all-rounder.
```
You stated that Nearest Neighbor has the "lowest performance cost", but two filters can't have the "lowest". Changed this one to say "lower" instead of lowest. Please let me know which one is actually the lowest.